api/instructor/instructorModel.js

```diff
@@ -1,14 +1,9 @@
 const db = require('../../data/db-config');
 
-const findInstructorCourses = async (profile_id) => {
-  return db('instructors')
-    .leftJoin(
-      'courses',
-      'instructors.instructor_id',
-      '=',
-      'courses.instructor_id'
-    )
-    .where('instructors.profile_id', profile_id);
+const findInstructorCourses = async (instructor_id) => {
+  return db('courses as c')
+    .leftJoin('instructors as i', 'i.instructor_id', 'c.course_id')
+    .where('c.instructor_id', instructor_id);
 };
```

Comment:
Love this change! it didn't make sense to pull from profile_id since instructor_id is what connects everything. And also the courses being the table you're pulling from and joining the instructor's table makes much more sense than how it was before.